content/files/Akeneo PIM API environment.postman_environment.json

```diff
@@ -0,0 +1,46 @@
+{
+  "id": "80352538-a8d0-7662-e092-72a218923ee4",
+  "name": "Akeneo PIM API environment",
+  "values": [
+    {
+      "enabled": true,
+      "key": "url",
+      "value": "http://my-favorite-pim.com",
+      "type": "text"
+    },
+    {
+      "enabled": true,
+      "key": "clientId",
+      "value": "1_4o4k6z2ydcisokg40ggssgogc8ws4kogoo888ocswkkckc0woo",
+      "type": "text"
+    },
+    {
+      "enabled": true,
+      "key": "secret",
+      "value": "3l6sukxg1f28so0ws4ss44cwsow0w44c4kgkwsosww88scckkc",
+      "type": "text"
+    },
+    {
+      "enabled": true,
+      "key": "username",
+      "value": "admin",
+      "type": "text"
+    },
+    {
+      "enabled": true,
+      "key": "password",
+      "value": "admin",
+      "type": "text"
+    },
+    {
```

Comment:
you can delete "base64ClientIdSecret" entry, it's generated on the fly (normally, I've tested that)